sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -52,9 +84,30 @@ add_sycl_plugin(unified_runtime
     # These below belong to Unified Runtime PI Plugin only
     "pi_unified_runtime.hpp"
     "pi_unified_runtime.cpp"
-    LIBRARIES
+  LIBRARIES
     Threads::Threads
     UnifiedRuntime-Headers
+    UnifiedRuntimeLoader-Headers
+    UnifiedRuntimeLoader
     LevelZeroLoader-Headers
     LevelZeroLoader
 )
+
+if (TARGET loader)
```

Comment:
I assume this `loader` target is from UR repo, right? Is it possible to name it so it is more clear?

Reply:
@smaslov-intel loader is the name being used at this moment in the UR. I could submit a PR to the UR to change the name.

Reply:
@smaslov-intel : used alias now, please see.